src/trading/ClaimProceeds.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controller.sol';
+import 'ROOT/reporting/Interfaces.sol';
+import 'ROOT/extensions/MarketFeeCalculator.sol';
+
+/**
+ * @title ClaimProceeds
+ * @dev This allows users to claim their money from a market by exchanging their shares 
+ */
+contract ClaimProceeds is Controlled {
+	uint256 private constant THREE_DAYS = 3 * 24 * 60 * 60;
```

Comment:
Solidity has a concept of units so this can be simplified to `= 3 days`.  Because of that, this variable is actually no longer useful so I recommend just changing the place in code its used to be `3 days` and deleting this variable.